llama-index-packs/llama-index-packs-rag-evaluator/llama_index/packs/rag_evaluator/base.py

```diff
@@ -64,6 +65,10 @@ def __init__(
         }
         self.eval_queue = deque(range(len(rag_dataset.examples)))
         self.prediction_dataset = None
+        if result_path is None:
+            self.result_path = "./"
+        else: 
+            self.result_path = result_path
```

Comment:
Nice, though do you think we can we do something a bit more robust?

Something like this: https://github.com/run-llama/llama_index/blob/7d3bb2b502c95e0677013ac935992d4cb1e36d9d/llama-index-core/llama_index/core/download/utils.py#L84

Reply:
Made it more robust now using Path from pathlib. Also creating dir if needed